app/experimenter/experiments/email.py

```diff
@@ -22,3 +23,46 @@ def send_review_email(experiment_name, experiment_url, needs_attention):
         [settings.EMAIL_REVIEW],
         fail_silently=False,
     )
+
+
+def send_intent_to_ship_email(experiment_id):
+
+    def make_url(uri):
+        return urljoin("https://{}".format(settings.HOSTNAME), uri)
+
+    experiment = Experiment.objects.prefetch_related(
+        "locales", "countries"
+    ).get(id=experiment_id)
+
+    experiment_url = make_url(experiment.experiment_url)
+
+    bug_url = settings.BUGZILLA_DETAIL_URL.format(id=experiment.bugzilla_id)
+
+    # Because that's how it's done in Experiment.population (property)
+    percent_of_population = f"{float(experiment.population_percent):g}%"
+
+    content = render_to_string(
+        "experiments/intent_to_ship.txt",
+        {
+            "experiment": experiment,
+            "experiment_url": experiment_url,
+            "bug_url": bug_url,
+            "percent_of_population": percent_of_population,
+            "locales": [str(l) for l in experiment.locales.all()],
+            "countries": [str(c) for c in experiment.countries.all()],
+        },
+    )
+    # Strip extra newlines from autoescape tag
+    content = content.strip() + "\n"
+
+    version = experiment.firefox_version
+    channel = experiment.firefox_channel
+    send_mail(
+        Experiment.INTENT_TO_SHIP_EMAIL_SUBJECT.format(
+            name=experiment.name, version=version, channel=channel
+        ),
+        content,
+        settings.EMAIL_SENDER,
+        [settings.EMAIL_REVIEW],
```

Comment:
This should be a new setting for a QA address called EMAIL_QA that we'll have to add to .env.sample and settings.py

Reply:
Actually, I'm a bit confused about this. Rereading the issue in question, it says:

- **To: release-drivers**: this is `EMAIL_REVIEW`, isn't it? (Honest question, I have no idea what these values are. We should probably improve `.env.sample`.)
- **Cc: Creator and anyone else we know about from experiment** -- I guess I can get `owner.email`? I don't think anything else is guaranteed to be an email.
- **From: experiments-review**: this is `EMAIL_SENDER`, isn't it?

Reply:
- no EMAIL_REVIEW is a different email for a now defunct mailing list.  Removing that setting is part of another ticket.
- Yes the only email we're guaranteed to have is the owner, they should be on the CC list
- Yes, EMAIL_SENDER is the correct email for the sender